README.md

```diff
@@ -38,23 +56,37 @@ This project adheres to the [Contributor Covenant code of conduct](./CODE_OF_CON
 
-Remember to watch the repo to get updates of all issues and comments.
+Remember to watch the repo to get updates on all issues and comments.
 
 ### I've got an idea to share
 
-If there is anyone with cool ideas of projects that doesn't have the time to create it themselves, [post it as an issue](https://github.com/open-source-ideas/open-source-ideas/issues/new) on this repo, mark it with the appropriate label and advertise it if you so please.
+If there is anyone with cool ideas for projects and doesn't have the time to create them, [post it as an issue](https://github.com/open-source-ideas/open-source-ideas/issues/new) on this repo, mark it with the appropriate label(s) and advertise it if you so please.
 
-Try to include as much information as needed to implement the project, and be open to answer questions if necessary. Add a label to the project whether it is estimated to be [`Little work`](https://github.com/open-source-ideas/open-source-ideas/labels/Little%20work), [`Medium work`](https://github.com/open-source-ideas/open-source-ideas/labels/Medium%20work) or [`Much work`](https://github.com/open-source-ideas/open-source-ideas/labels/Much%20work). Also try to answer the issue template as correct as possible. If the project is perfect for first-timers, label the Issue as [`First Contribute`](https://github.com/open-source-ideas/open-source-ideas/labels/First%20Contribute).
+Try to include as much information as needed to implement the project, and be open to answering questions if necessary. Add a label to the project for whether it is estimated to be [`Little work`](https://github.com/open-source-ideas/open-source-ideas/labels/Little%20work), [`Medium work`](https://github.com/open-source-ideas/open-source-ideas/labels/Medium%20work) or [`Much work`](https://github.com/open-source-ideas/open-source-ideas/labels/Much%20work).
+<br/>
+Also, try to answer the issue template as correctly as possible. If the project is perfect for first-timers, label the issue as [`First Contribute`](https://github.com/open-source-ideas/open-source-ideas/labels/First%20Contribute).
+<br/>
+Finally, try to label the issue with the most appropriate tech categor(y|ies) to attract excited, motivated developers to your idea! (see list above)
 
 ### I want to create a project
 
-If you are Open Source-curious or just looking for your next project, you can look through the Issue-list and pick one of your liking. If you find anything of your liking, you can join the issue by commenting on it. If there are people who want to cooperate, that's fantastic. Just make a comment and the person who joined it first can answer.
+If you are Open Source-curious or just looking for your next project, you can look through the Issue-list and pick one of your liking. If anything captures your imagination, you can join the issue by commenting on it. If there are people who want to cooperate, that's fantastic. Just submit a comment and the person who created the issue can answer.
```

Comment:
Changed "Just make a comment and the person who joined it first can answer." _to_ ""Just submit a comment and the person who created the issue can answer."...I assume this is what you mean? Thought it was a little clearer; initially I thought the person who first commented can answer, which I don't believe is the case?